pom.xml

```diff
@@ -127,7 +127,7 @@
     <java.min.version>${compileSource}</java.min.version>
     <maven.min.version>3.3.3</maven.min.version>
     <rename.offset>org.apache.hbase.thirdparty</rename.offset>
-    <protobuf.version>3.9.2</protobuf.version>
+    <protobuf.version>3.11.1</protobuf.version>
```

Comment:
Any reason not going to 3.11.2 directly?
https://github.com/protocolbuffers/protobuf/releases/tag/v3.11.2

Reply:
It was not up in the mvn repos when I was at this last night https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java ... and no java commits in 3.11.2 anyways: https://github.com/protocolbuffers/protobuf/compare/v3.11.2...3.11.x